GameServerCore/Enums/SpellDataFlags.cs

```diff
@@ -0,0 +1,50 @@
+﻿using System;
+
+namespace GameServerCore.Enums
+{
+    /// <summary>
+    /// Enumerator containing all(?) possible spell flags found in spell data. Used in determining how a spell functions.
+    /// </summary>
+    /// TODO: Verify if this is finished.
+    [Flags]
+    public enum SpellDataFlags
+    {
+        AutoCast = 1 << 1,
+        InstantCast = 1 << 2,
+        PersistThroughDeath = 1 << 3,
+        NonDispellable = 1 << 4,
+        NoClick = 1 << 5,
+        AffectImportantBotTargets = 1 << 6,
+        AllowWhileTaunted = 1 << 7,
+        NotAffectZombie = 1 << 8,
+        AffectUntargetable = 1 << 9,
+        AffectEnemies = 1 << 10,
+        AffectFriends = 1 << 11,
+        AffectNeutral = 1 << 14,
+        // 4000 + C00: 4C00
+        AffectAllSides = AffectNeutral + 3 * (1 << 10),
```

Comment:
Is it the same as `AffectNeutral +AffectFriends +AffectEnemies `?

Reply:
Correct, the total = (1 << 10) + (1 << 11) + (1 << 14).
I suppose I could change it to (AffectEnemies + AffectFriends + AffectNeutral) if you'd like.